smoke-test/metric_service_smoke_test.ts

```diff
@@ -12,9 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-'use strict';
-
-const {describe, it} = require('mocha');
+import {describe, it} from 'mocha';
```

Comment:
This smoke test was not hand written.  It was a weird generated thing that only some of the old monolith gapics generated.  If we aren't sure about value, removing is ok.  This module has decent sample test coverage.

Reply:
Thanks. Removed.